test/algorithms/simulation/sidb/determine_physically_valid_parameters.cpp

```diff
@@ -0,0 +1,163 @@
+//
+// Created by Jan Drewniok on 03.05.24.
+//
+
+#include <catch2/catch_test_macros.hpp>
+
+#include "utils/blueprints/layout_blueprints.hpp"
+
+#include <fiction/algorithms/simulation/sidb/determine_physically_valid_parameters.hpp>
+#include <fiction/algorithms/simulation/sidb/operational_domain.hpp>
+#include <fiction/algorithms/simulation/sidb/sidb_simulation_parameters.hpp>
+#include <fiction/layouts/coordinates.hpp>
+#include <fiction/technology/cell_technologies.hpp>
+#include <fiction/technology/charge_distribution_surface.hpp>
+#include <fiction/technology/sidb_charge_state.hpp>
+#include <fiction/types.hpp>
+#include <fiction/utils/phmap_utils.hpp>
+
+using namespace fiction;
+
+TEST_CASE("Determine physical parameters for cds of SiQAD Y-shaped AND gate, 10 input combination",
```

Comment:
```suggestion
TEST_CASE("Determine physical parameters for CDS of SiQAD Y-shaped AND gate, 10 input combination",
```
Capitalize CDS everywhere